packages/cli/src/commands/baseCommand.ts

```diff
@@ -9,9 +19,9 @@ export abstract class BaseCommand extends Command {
   static coreCommand = false
   static hidden = true
 
-  protected init (): Promise<void> {
+  protected async init (): Promise<void> {
     // TODO: find a better way to mock the CLI version for E2E tests.
-    api.defaults.headers['x-checkly-cli-version'] = process.env.CHECKLY_CLI_VERSION ?? this.config.version
+    api.defaults.headers['x-checkly-cli-version'] = process.env.CHECKLY_CLI_VERSION ?? await getLatestVersion()
     return super.init()
```

Comment:
This is gonna cause every cli version to report the wrong version. Just pass the version as an env var during the tests please

Reply:
Good catch! Thanks.

Reply:
Reverted. `oclif` version (`this.config.version`) is used again. @umutuzgur , my concern is that if you are using you local env without configuring correct version in the `CHECKLY_CLI_VERSION` environment variable, the CLI will use an 'old' version and the API request header will be outdated. Currently, we have two different behaviors between `4.0.8` and `>4.0.8` for private-locations assignments. This could result in some annoying results (i.e. PLs assigments will be removed if you are using `>= 4.0.9` source code without `CHECKLY_CLI_VERSION` set). I just updated the `README.md` and `CONTRIBUTING.md`.